### PR TITLE
added complete project

### PR DESCRIPTION
**Pull request by 1khansaad**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+</head>
+<body>
+    <div id="container">
+        <div class="one">
+            <img src="logo.png" alt="one-plus-logo" width="80px">
+            <h2> The spirit of the OnePlus family.</h2>
+            <p>It's the belief that we can always push harder and go a little further. The words on this shirt motivate us to push our limits every day, just as we think they can inspire you.</p>
+        </div>
+        <div class="two">
+            <img src="phone-1.png" alt="" width="300px">
```
Comment (Abbas4tech) [inline index.html]: Add `alt` attribute

File: index.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+            <h2>Smart Phones</h2>
+        </div>
+        <div class="three">
+            <img src="neckband.png" alt="" width="130px">
+            <h2>OnePlus Bullets</h2>
+        </div>
+        <div class="four">
+            <img src="pods.png" alt="" width="250px">
```
Comment (Abbas4tech) [inline index.html]: Add `alt` attribute

File: style.css [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+  margin-top: 40px;
+}
+.seven p {
+  font-size: 20px;
+}
+@media screen and (max-width: 768px) {
+  #container {
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-auto-rows: minmax(200px, auto);
```
Comment (Abbas4tech) [inline style.css:76]: Can you explain how values `minimax(200px, auto)` effect

**Review by Abbas4tech — CHANGES_REQUESTED** [review]

**Review by Tanvir04khan — APPROVED** [review]

File: index.html [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+            <img src="logo.png" alt="one-plus-logo" width="80px">
+            <h2> The spirit of the OnePlus family.</h2>
+            <p>It's the belief that we can always push harder and go a little further. The words on this shirt motivate us to push our limits every day, just as we think they can inspire you.</p>
+        </div>
+        <div class="two">
+            <img src="phone-1.png" alt="" width="300px">
+            <h2>Smart Phones</h2>
+        </div>
+        <div class="three">
+            <img src="neckband.png" alt="" width="130px">
```
Comment (mujeeb100) [inline index.html]: ALSO HERE TO ADD ALT ATTRIBUTE

File: style.css [hunk 5]
```diff
@@ -0,0 +1,105 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+body {
+  text-align: center;
+
```
Comment (mujeeb100) [inline style.css:8]: REMOVE THE SPACE IN CSS FILE..

**Review by anask601 — APPROVED** [review]
LGTM

**Review by mujeeb100 — CHANGES_REQUESTED** [review]
CHANGES REQUIRED

**Comment by Rehan-Khan-Engineer** [comment]
I think Responsiveness could be improved but overall great efforts @1khansaad. Its looking pleasing.

**Review by Rehan-Khan-Engineer — APPROVED** [review]

File: index.html (hunk 2 above)
Comment (1khansaad) [reply index.html]: done bhai @Abbas4tech

File: style.css (hunk 3 above)
Comment (1khansaad) [reply style.css:76]: if the content inside the grid is less than 200px so it takes the min-height of grid as 200px in this case there is no role of auto height but if content inside the grid is more than 200px so takes the content height. @Abbas4tech

**Review by Tauqeer-Ahmed-99 — APPROVED** [review]

**Pull request merged**